controller/project/manager.py

```diff
@@ -274,11 +274,7 @@ def __no_huddle_id(data_id: str) -> bool:
 
 def __get_crowd_label_is_data(project_id: str, is_id: str) -> Dict[str, str]:
     information_source_item = information_source.get(project_id, is_id)
-    if (
-        not information_source_item
-        or information_source_item.type
-        != enums.InformationSourceType.CROWD_LABELER.value
-    ):
+    if not information_source_item:
```

Comment:
i think the whole function can be removed

Reply:
yes it can be, 
this is connected with the roles and slices type so i though to remove it with this point